common/src/main/resources/assets/minecraft_access/lang/en_us.json

```diff
@@ -205,12 +206,11 @@
   "minecraft_access.messages.accessibility_permission_not_granted": "The accessibility permission has not been granted, so mouse simulation will not work. Check docs slash setup slash MacOS dot md in the Github repository for instructions on how to enable it.",
   "minecraft_access.other.animal_variant_format": "{variant} {animal}",
   "minecraft_access.other.biome": "%s biome entered",
-  "minecraft_access.other.craftable": "Craftable",
   "minecraft_access.other.entity_with_equipments": "{entity} with {equipments}",
   "minecraft_access.other.facing_direction": "Facing %s",
   "minecraft_access.other.negative": "negative %s",
-  "minecraft_access.other.not_craftable": "Not Craftable",
```

Comment:
`minecraft_access.other.craftable` and `minecraft_access.other.not_craftable` are still used.

https://github.com/minecraft-access/minecraft-access/blob/ac9a286c752aa164bdbd03e1474ff611023f5d21/common/src/main/java/org/mcaccess/minecraftaccess/mixin/RecipeButtonMixin.java#L42-L45